### ISSUE TITLE
Readiness Checklist step 6 update proposal

### ISSUE THREAD

**Issue by FabrizioMoggio** [camaraproject/Commonalities#12]
With a reference to Issue camaraproject/WorkingGroups#139 (https://github.com/camaraproject/WorkingGroups/issues/139).

I suggest to discuss about step 6 of the Readiness Check List (https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-Readiness-Checklist.md)

The discussion is driven by the goal of not slowing dawn the possibility for an Operator to promote or sell an API in its marketplace. Step 6 requires two Operators to have implemented and tested an API. Because of this step, it can happen that a stable API can not be marked as v1.0.0 because a second Operator has not yet implemented it.

A **proposal** could be a modification of step 6 having j**ust one Operator as mandatory** and a **second Operator or a CAMARA member as recommended.**

We could also evaluate the possibility for an API to be promoted and published on a marketplace even if in **Beta version**. A Beta version is expected to have possible improvements with minor releases while a "full version" (e.g. v1.0.0) is expected to be more stable. 

The idea is to gather, in this discussion, the opinion from the group and to **present a possible solution to the Governance WG.** this because it is a marketing decision rather than a technical one.

**Comment by jordonezlucena** [comment]
Dear all,
This is a long-standing discussion. I see pros (as raised by @FabrizioMoggio) and cons (as noted by @hdamker, probably also shared by GSMA in their goal to increase operator adoption). 
GSMA has drafted a WoW between Product WS and CAMARA, with the aim to agree on an agile interaction model for collaboration between both tracks. Among other things, this WoW proposes how both tracks can work together to ensure release cadence of for APIs (important to manage expectations of developers and operator with regards to **when** new API releases will be available, and **with which scope**). I think this proposal is closely related to the versioning topic proposed in this issue. 
GSMA Open Gateway Product WS will be presenting this WoW in next TSC meeting (7th Sept), in order to get feedback from TSC and finalise on details (e.g. periodicity of API releases). My suggestion is to escalate this issue to the TSC, leaving them to use i) this issue and ii) WoW, as input to decide way forward

**Comment by FabrizioMoggio** [comment]
@jordonezlucena I agree with you, if we can't find a solution we can escalate to the TSC. Anyway, just to verify if we are all on the same page, **my proposal is** to follow @hdamker suggestion to have V1.0.0 according to the current step six (two Operators, I see the value for GSMA) and to add to step six a note regarding the Beta version: if an API has completed all the required steps from the checklist but it is just implemented by one Operator, it can be anyway published on the market as Beta version.

**Comment by jordonezlucena** [comment]
@FabrizioMoggio @hdamker: we should steer all the discussion regarding when/how to achieve 1.0 through one single issue - otherwise, it is verify difficult to keep track of progress. I say this because I've just come up with a quite similar issue in Governance, see [#85](https://github.com/camaraproject/Governance/issues/85). I guess it falls within the scope of TSC.

**Comment by FabrizioMoggio** [comment]
@shilpa-padgaonkar how can we bring this discussion to TSC? I mean, what I need to do :-)

**Comment by shilpa-padgaonkar** [comment]
@FabrizioMoggio In the last TSC call it was agreed that I would label all issues in commonalities which are somehow relevant for release mgmt concept, under the label release mgmt so that they can be addressed once the topic of release mgmt (E2E) gets started/discussed within TSC in the next/upcoming TSC calls.
The TSC will reach out to you as soon as your point is on the agenda.
